server/commands/documentCreator.ts

```diff
@@ -147,6 +147,10 @@ export default async function documentCreator({
     }
   );
 
+  if (collectionId) {
+    publish = true;
+  }
```

Comment:
Should we do this on the client instead? Silently turning non-publish into publish in the app may be a bit subtle.

Reply:
Yea agreed, it would be nice to keep the API backwards compatible as well

Reply:
Sure thing